nifi-nar-bundles/nifi-couchbase-bundle/nifi-couchbase-processors/src/test/java/org/apache/nifi/processors/couchbase/TestPutCouchbaseKey.java

```diff
@@ -268,7 +268,7 @@ public void testInputFlowFileUuid() throws Exception {
 
         ArgumentCaptor<RawJsonDocument> capture = ArgumentCaptor.forClass(RawJsonDocument.class);
         verify(bucket, times(1)).upsert(capture.capture(), eq(PersistTo.NONE), eq(ReplicateTo.NONE));
-        assertEquals(uuid, capture.getValue().id());
+        // assertEquals(uuid, capture.getValue().id());
```

Comment:
If you drop all of these commented lines, I'll go ahead and merge this.

Reply:
Do you want me to revert the changes made to the unit tests?

Reply:
No, I want you to completely delete the line of code instead of commenting it.

Reply:
Done